ci/images/ci-runner/hack/bin/copy-plnsvc-code.sh

```diff
@@ -5,7 +5,9 @@ set -o pipefail
 set -x
 
 echo "Copy source code of pipeline service to the ci-runner container"
+kubectl exec -n default -it ci-runner -- bash -c "rm -rf /source/*"
```

Comment:
I'm expecting this to fail on the very first run, thus ending the PLR in failure. Check for the folder existence before deletion.

Same for the next line.

Reply:
I am confusing why you are expecting this to fail on the very first run, because directory `/source` already is there , it always succeed no matter if there are files or not under  `/source/`